python/lsst/meas/extensions/gaap/_gaap.py

```diff
@@ -296,18 +317,25 @@ def measure(self, measRecord: lsst.afw.table.SourceRecord, exposure: afwImage.Ex
                 errorCollection[str(sF)] = error
                 continue
 
-            for sigma in self.config.sigmas:
+            for sigma in sigmas:
                 baseName = self.ConfigClass.getGaapResultName(sF, sigma, self.name)
-                if targetSigma >= sigma:
+                try:
+                    aperSigma2 = sigma**2 - targetSigma**2
+                    aperShape = afwGeom.Quadrupole(aperSigma2, aperSigma2, 0.0, normalize=True)
+                    fluxScaling = 0.5*sigma**2/aperSigma2  # Eq. A16 of Kuijken et al. (2015)
+                except TypeError:
+                    # PSF photometry case
+                    aperSigma2 = targetSigma**2
+                    aperShape = afwGeom.Quadrupole(aperSigma2, aperSigma2, 0.0)
+                    fluxScaling = 1.0
+                except InvalidParameterError:
+                    # This implies sigma < targetSigma
```

Comment:
Similar to above: why not explicitly check for `sigma >= targetSigma`?

Reply:
OK, I have a part if/else and try/except as a compromise. I am not explicitly using `sigma >= targetSigma` conditions, because in the next ticket, for elliptical apertures, it is more complicated and I let `afw` take care of it. The validity of the config parameters is questionable, as it depends on the data as well and it is not easy for the user to know what the valid values of `sigmas` are. So it makes sense to raise an exception and handle here instead of if/else statements.